subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -79,6 +79,12 @@
             "changes": [
                 "Method has been removed"
             ]
+        },
+        {
+            "type": "org.gradle.api.tasks.compile.GroovyCompile",
+            "member": "Method org.gradle.api.tasks.compile.GroovyCompile.getFeaturePreviews()",
+            "acceptation": "Injected property",
```

Comment:
Is this really necessary? I thought we already exclude injected properties: https://github.com/gradle/gradle/blob/a70f1ae699f12b5479dc969314a5c6f3518269dd/buildSrc/subprojects/binary-compatibility/src/main/groovy/org/gradle/binarycompatibility/rules/IncubatingMissingRule.java#L38

If it is, we probably should fix it.

Reply:
Fixed in another PR: https://github.com/gradle/gradle/pull/9916